security.md

```diff
@@ -72,61 +72,137 @@ Building Gems
 ### Sign with: `gem cert`
 
 1) Create self-signed gem cert
-
+    ```shell
```

Comment:
The enumerated list was not proper markdown, and after a few syntax changes, which should also allow language-specific code highlighting, it is now proper markdown.